modules/plugin/imagemosaic/pom.xml

```diff
@@ -207,20 +207,4 @@
   </dependencies>
 
 
-  <build>
```

Comment:
I have built the imageMosaic module on Windows (on top of your aaime:replaceSerializableRenderedImage) and I had no issues.